src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceManagerService.java

```diff
@@ -87,19 +87,14 @@ public Project getProject(String googleProjectId) {
   }
 
   public void deleteProject(String googleProjectId) {
-    // Don't actually delete the project if we are reusing projects!
-    if (resourceConfiguration.getAllowReuseExistingProjects()) {
-      logger.info("Reusing projects: skipping delete of {}", googleProjectId);
-    } else {
-      try {
-        CloudResourceManager resourceManager = cloudResourceManager();
-        CloudResourceManager.Projects.Delete request =
-            resourceManager.projects().delete(googleProjectId);
-        // the response will be empty if the request is successful in the delete
-        request.execute();
-      } catch (IOException | GeneralSecurityException e) {
-        throw new GoogleResourceException("Could not delete project", e);
-      }
+    try {
+      CloudResourceManager resourceManager = cloudResourceManager();
+      CloudResourceManager.Projects.Delete request =
+          resourceManager.projects().delete(googleProjectId);
+      // the response will be empty if the request is successful in the delete
+      request.execute();
```

Comment:
Seems like a good opportunity for `var`. 
```suggestion
      var resourceManager = cloudResourceManager();
      var request = resourceManager.projects().delete(googleProjectId);
      // the response will be empty if the request is successful in the delete
      request.execute();
```
Or you could write it in one line:
```suggestion
      // the response will be empty if the request is successful in the delete
      cloudResourceManager().projects().delete(googleProjectId).execute();
```

Reply:
This seemed to break the project delete code (at least, the one test run failed on "Unable to delete project"). Given that it's outside of the scope of my changes, I'm going to leave it as is.